stdlib/array.mli

```diff
@@ -230,39 +253,39 @@ val sort : ('a -> 'a -> int) -> 'a array -> unit
    compare as equal, a positive integer if the first is greater,
    and a negative integer if the first is smaller (see below for a
    complete specification).  For example, {!Stdlib.compare} is
-   a suitable comparison function.  After calling [Array.sort], the
+   a suitable comparison function, provided there are no floating-point
+   NaN values in the data.  After calling [sort], the
    array is sorted in place in increasing order.
-   [Array.sort] is guaranteed to run in constant heap space
+   [sort] is guaranteed to run in constant heap space
    and (at most) logarithmic stack space.
 
    The current implementation uses Heap Sort.  It runs in constant
    stack space.
 
    Specification of the comparison function:
    Let [a] be the array and [cmp] the comparison function.  The following
-   must be true for all [x], [y], [z] in [a] :
+   must be true for all x, y, z in a :
```

Comment:
Why this change? It looks like we normally use brackets for variable names...

Reply:
I have no idea. Will revert.